### ISSUE TITLE
Dimension.get('screen') returns window width and height, not a device screen width and height in RN v0.63.4

### ISSUE THREAD

**Issue by oleksandr-dziuban** [facebook/react-native#30577]
## Description
Hi, issue with Dimension change event listener is partially fixed in 0.63.4, but I can confirm, we have problems now with Dimensions.get('screen') on iOS and Android in react-native@0.63.4.
Width and height of the screen are changing in the same way like a Dimensions.get('window'). This is Incorrect behaviour. Device screen width/height should be the same when we change device orientation. Only the window width/height should be changed according to available rendered window.

## React Native version:
v0.63.4

## Steps To Reproduce
Use this hook in any react native component in basic app boilerplate:

```jsx
import { useEffect, useCallback } from 'react';
import { Dimensions } from 'react-native';

// Hook to handle orientation in functional components
const useOrientation = () => {
  const handleOrientationChange = useCallback(() => {
    const {width: screenWidth, height, screenHeight} = Dimensions.get('screen');
    const {width: windowWidth, height, windowHeight} = Dimensions.get('window');

    console.log(windowWidth, windowHeight); // Correct - changes according to window width / height
    console.log(screenWidth, screenHeight); // Incorrect - changes according to window width / height, but should not be changed on orientation change as device screen width / height is the same
  }, []);

  useEffect(() => {
    Dimensions.addEventListener('change', handleOrientationChange);

    return () => {
      Dimensions.removeEventListener('change', handleOrientationChange);
    };
  }, [handleOrientationChange]);

  return orientationParams;
};

export default useOrientation;
```

## Expected Results
Dimensions.get('screen') params should not be changed on orientation change on iOS/Android.

**Comment by AND-GORNIY** [comment]
> ## Description
> Hi, issue with Dimension change event listener is partially fixed in 0.63.4, but I can confirm, we have problems now with Dimensions.get('screen') on iOS and Android in react-native@0.63.4.
> Width and height of the screen are changing in the same way like a Dimensions.get('window'). This is Incorrect behaviour. Device screen width/height should be the same when we change device orientation. Only the window width/height should be changed according to available rendered window.
> 
> ## React Native version:
> v0.63.4
> 
> ## Steps To Reproduce
> Use this hook in any react native component in basic app boilerplate:
> 
> ```js-jsx
> import { useEffect, useCallback } from 'react';
> import { Dimensions } from 'react-native';
> 
> // Hook to handle orientation in functional components
> const useOrientation = () => {
>   const handleOrientationChange = useCallback(() => {
>     const {width: screenWidth, height, screenHeight} = Dimensions.get('screen');
>     const {width: windowWidth, height, windowHeight} = Dimensions.get('window');
> 
>     console.log(windowWidth, windowHeight); // Correct - changes according to window width / height
>     console.log(screenWidth, screenHeight); // Incorrect - changes according to window width / height, but should not be changed on orientation change as device screen width / height is the same
>   }, []);
> 
>   useEffect(() => {
>     Dimensions.addEventListener('change', handleOrientationChange);
> 
>     return () => {
>       Dimensions.removeEventListener('change', handleOrientationChange);
>     };
>   }, [handleOrientationChange]);
> 
>   return orientationParams;
> };
> 
> export default useOrientation;
> ```
> 
> ## Expected Results
> Dimensions.get('screen') params should not be changed on orientation change on iOS/Android.

If you need static width and height of screen you can simple use consts before your useOrientation. 

rough example
```
const screenDimensions = Dimension.get('screen');
const screenWidth  =  screenDimensions.height > screenDimensions.width ?  screenDimensions.width : screenDimensions.height
const screenHeight = screenDimensions.height < screenDimensions.width ?  screenDimensions.width : screenDimensions.height
```
You can`t fully trust prop 'window' to listen orientation due to fact multitask split view on IOS.

**Comment by oleksandr-dziuban** [comment]
> > ## Description
> > Hi, issue with Dimension change event listener is partially fixed in 0.63.4, but I can confirm, we have problems now with Dimensions.get('screen') on iOS and Android in react-native@0.63.4.
> > Width and height of the screen are changing in the same way like a Dimensions.get('window'). This is Incorrect behaviour. Device screen width/height should be the same when we change device orientation. Only the window width/height should be changed according to available rendered window.
> > ## React Native version:
> > v0.63.4
> > ## Steps To Reproduce
> > Use this hook in any react native component in basic app boilerplate:
> > ```
> > import { useEffect, useCallback } from 'react';
> > import { Dimensions } from 'react-native';
> > 
> > // Hook to handle orientation in functional components
> > const useOrientation = () => {
> >   const handleOrientationChange = useCallback(() => {
> >     const {width: screenWidth, height, screenHeight} = Dimensions.get('screen');
> >     const {width: windowWidth, height, windowHeight} = Dimensions.get('window');
> > 
> >     console.log(windowWidth, windowHeight); // Correct - changes according to window width / height
> >     console.log(screenWidth, screenHeight); // Incorrect - changes according to window width / height, but should not be changed on orientation change as device screen width / height is the same
> >   }, []);
> > 
> >   useEffect(() => {
> >     Dimensions.addEventListener('change', handleOrientationChange);
> > 
> >     return () => {
> >       Dimensions.removeEventListener('change', handleOrientationChange);
> >     };
> >   }, [handleOrientationChange]);
> > 
> >   return orientationParams;
> > };
> > 
> > export default useOrientation;
> > ```
> > 
> > 
> > ## Expected Results
> > Dimensions.get('screen') params should not be changed on orientation change on iOS/Android.
> 
> If you need static width and height of screen you can simple use consts before your useOrientation.
> 
> rough example
> 
> ```
> const screenDimensions = Dimension.get('screen');
> const screenWidth  =  screenDimensions.height > screenDimensions.width ?  screenDimensions.width : screenDimensions.height
> const screenHeight = screenDimensions.height < screenDimensions.width ?  screenDimensions.width : screenDimensions.height
> ```
> 
> You can`t fully trust prop 'window' to listen orientation due to fact multitask split view on IOS tablets.

Hi, this hack is already added :) Main idea of this issue is to restore previous behavior for Dimensions. Before 0.63.3 it was OK, not event in 0.63.4 it's still incorrect.
